packages/openactive-integration-tests/test/features/core/amending-order-quote/implemented/amend-c1-test.js

```diff
@@ -59,26 +60,31 @@ FeatureHelper.describeFeature(module, {
       await chakram.wait();
     });
 
-    (new GetMatch({
-      state, flow, logger, orderItemCriteria,
-    }))
-      .beforeSetup()
-      .successChecks()
-      .validationTests();
-    const c1 = (new C1({
-      state, flow, logger,
-    }))
-      .beforeSetup()
-      .successChecks()
-      .validationTests();
+    describe('Get Opportunity Feed Items', () => {
```

Comment:
I've updated this existing test to use the same describe-block-for-each-stage-pattern as other tests.

I assume that this pattern exists in order to keep the test output consistent

I initially didn't want to do this because I thought it might be possible to only use describe() blocks to describe tests that could feasibly be run in parallel. I belive this is usually best practice (so also thinks this person: https://stackoverflow.com/a/39856222) as it allows you to turn up test parallelization if tests become slow and will also allow porting to other test frameworks that do not have synchronous guarantuees about describe blocks.

However, I realised that that would not really be possible without changing the architecture quite a lot (e.g. the functions that create describe() / it() / afterAll() / etc blocks would all need to work differently). And changing the architecture would mean that the test output would not be anywhere near as consistent (not without a lot of effort anyway), so I've decided to go with the architecture rather than against.

Maybe we can think about the parallelization question later. Much later after core tests have been implemented :)

Reply:
Right. That's helpful to know. Thanks 🐩